### PR TITLE
use storage token instead of job token

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/PS-3726
Counterpart to https://github.com/keboola/job-queue-daemon/pull/322

### CODE REVIEW

File: config/services.yaml [hunk 1]
```diff
@@ -74,7 +75,8 @@ services:
     Keboola\JobQueueInternalClient\Client:
         arguments:
             $internalQueueApiUrl: '%env(JOB_QUEUE_URL)%'
-            $internalQueueToken: '%env(JOB_QUEUE_TOKEN)%'
+            $internalQueueToken: ~
+            $storageApiToken: '%env(STORAGE_API_TOKEN)%'
```
Comment (odinuv) [inline config/services.yaml:79]: This is the actual change - STORAGE_API_TOKEN env is added in https://github.com/keboola/job-queue-daemon/pull/322/files

Comment (pivnicek) [reply config/services.yaml:79]: sorry, does `~` mean null or emtpy string?  I don't know the syntax

Comment (odinuv) [reply config/services.yaml:79]: yes https://js.educative.io/answers/how-to-represent-null-values-in-yaml

Comment (pivnicek) [reply config/services.yaml:79]: cool, thanks 🍻

**Review by pivnicek — APPROVED** [review]
LGTM

**Pull request merged**